### ISSUE TITLE
Train on qald-9 with wikidata queries

### ISSUE THREAD

**Issue by makevin23** [rashad101/SGPT-SPARQL-query-generation#5]
Since I'm working on qald-9 with wikidata SPARQL queries, I want to test the performance of SGPT on it. According to the paper, SGPT should work with wikidata SPARQL queries, since they are in LC-QuAD and vquanda data set. 

I checked the code and found out that the script only read the value in "fil_sparql", so I replaced DBpedia SPARQL queries with Wikidata SPARQL queries. Like this:
![image](https://github.com/rashad101/SGPT-SPARQL-query-generation/assets/50887036/54732697-d375-4fd7-a0d1-6eca0fa03d03)

Then, I trained it using the same setting as for training with DBpedia queries. While training the loss kept to be high (about 2~3). 
In the end, I ran evaluation on the new model, the predictions look like this:
![image](https://github.com/rashad101/SGPT-SPARQL-query-generation/assets/50887036/4ba06027-cc75-444b-bcf0-4bd24772625c)
The predicted SPARQL queries contain many syntax errors and only a few entities and properties are correct. 

I am not sure what I did wrong with this system. Can you give me some instructions to it, how can I do it properly?

I've also tried to train on LC-QuAD and vquanda, but it failed since the same issue in #2 .

**Comment by rashad101** [comment]
Hi makevin23,
training on a particular dataset is sensitive to hyperparameters. The training error reported in issue #2 is resolved now. I'll update the readme with the suitable hyper-parameters soon.

Thanks for your understanding.
